src/EventStore.Core/Services/Storage/StorageScavenger.cs

```diff
@@ -96,8 +96,11 @@ private void Scavenge(ClientMessage.ScavengeDatabase message)
 
                     var scavenger = new TFChunkScavenger(_db, _ioDispatcher, _tableIndex, _readIndex, scavengeId,
                                                                                  _nodeEndpoint, unsafeIgnoreHardDeletes: _unsafeIgnoreHardDeletes);
-                    spaceSaved = scavenger.Scavenge(_alwaysKeepScavenged, _mergeChunks);
-                    result = ClientMessage.ScavengeDatabase.ScavengeResult.Success;
+                    int failedCount;
```

Comment:
@hayley-jean here is where the StorageScavenger received the number of failure and set the Warning status